kserve/runtimes/24.05-llama3-8b-instruct.yaml

```diff
@@ -50,5 +43,5 @@ spec:
   volumes:
   - emptyDir:
       medium: Memory
-      sizeLimit: 128Gi
-    name: dshm
\ No newline at end of file
+      sizeLimit: 16Gi
```

Comment:
Is this the value that is recommended by engineering? I haven't seen anything published in a sizing guide yet.

Reply:
https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#run-inference